tests/patch-packagefile.nix

```diff
@@ -4,20 +4,28 @@ let
   i = npmlock2nix.internal;
 in
 (testLib.runTests {
-  testTurnsGitHubRefsToStorePaths = {
+  testTurnsGitHubRefsToWildcards = {
     expr =
       let
         leftpad = (npmlock2nix.internal.patchPackagefile noGithubHashes ./examples-projects/github-dependency/package.json).dependencies.leftpad;
       in
-      lib.hasPrefix ("file://" + builtins.storeDir) leftpad;
+      leftpad == "*";
```

Comment:
In this case we should probably just set expected to `"*"` because then we get proper diffs if things fail.